.github/workflows/int-and-connected-test-run.yml

```diff
@@ -1,103 +1,42 @@
 name: Unit, Smoke, Connected and Integration tests
-env:
-  K8_CLUSTER: 'integration-master'
-  # This must be defined for the bash redirection
-  GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
-  # This must be defined for the bash redirection
-  GOOGLE_SA_CERT: 'jade-dev-account.pem'
-  # This will make test logs not use JSON
-  TDR_LOG_APPENDER: 'Console-Standard'
-  # The default Azure credentials to use to authenticate
-  AZURE_CREDENTIALS_APPLICATIONID: 0e29ec36-04e8-44d5-ae7c-50dc15135571
-  AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
-  JADE_USER_EMAIL: connected-tdr-user@notarealemail.org
-  RBS_CLIENT_CREDENTIAL_FILE_PATH: rbs-tools-sa.json
-  AZURE_CREDENTIALS_SECRET: ${{ secrets.AZURE_CREDENTIALS_SECRET }}
-  AZURE_SYNAPSE_SQLADMINUSER: ${{ secrets.AZURE_SYNAPSE_SQLADMINUSER }}
-  AZURE_SYNAPSE_SQLADMINPASSWORD: ${{ secrets.AZURE_SYNAPSE_SQLADMINPASSWORD }}
-  AZURE_SYNAPSE_WORKSPACENAME: tdr-snps-int-east-us-ondemand.sql.azuresynapse.net
-  CACHE_PATHS: |
-    build/jacoco
-    build/reports
-    build/spotless
-    build/test-results
-    build/jacocoHtml
 on:
-  pull_request:
-    branches:
-      - develop
-    paths:
-      - '!*'
-      - 'src/**'
-      - 'gradle**'
-      - 'gradle/**'
-      - '**.gradle'
-      - 'Dockerfile'
-      - 'datarepo-clienttests/**'
-      - '.github/workflows/int-and-connected-test-run.yml'
-      - '.swagger-codegen-ignore'
   workflow_dispatch: {}
+  pull_request:
   schedule:
     - cron: '0 4 * * *' # run at 4 AM UTC, 12PM EST.
+# do not allow concurrent runs of this workflow on the same branch
 concurrency:
-  # Don't run this workflow concurrently on the same branch
   group: ${{ github.workflow }}-${{ github.ref }}
-  # For PRs, don't wait for completion of existing runs, cancel them instead
   cancel-in-progress: ${{ github.event_name == 'pull_request' }}
 jobs:
-  test_check:
-    name: "Checkout, verify and run unit tests"
-    outputs:
-      job-status: ${{ job.status }}
+  test_unit:
+    name: Run unit tests
+    runs-on: ubuntu-latest
     timeout-minutes: 60
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    if: "!contains( github.event.pull_request.labels.*.name, 'skip-ci')"
-    runs-on: ${{ matrix.os }}
-    ## skips if pr label is 'skip-ci'
-    # run a local Postgres container in Docker for the basic check tests
-    services:
-      postgres:
-        image: postgres:11
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: postgres
-          POSTGRES_DB: postgres
-        ports:
-          - 5432:5432
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
```

Comment:
Reviewer note: we don't need Postgres for unit tests. Makes sense!

Reply:
Oh I'm a bit surprised by this!  Our unit tests contain a number of embedded database tests, do those need a Postgres?

Reply:
They create their own Postgres instances using `testcontainers`.

Reply:
Rad!  That's great.

Reply:
> Makes sense

It does today but before the embedded db support was added, all Dao unit tests required postgres. In many (most?) other terra service projects, Dao unit tests depend on postgres, so that behavior isn't uncommon.

Reply:
For a point of reference, we migrated Consent to postgres testcontainer-based unit tests in March 2020.

Reply:
This is interesting -- it makes more sense to me that we'd have this Postgres dependency left lying around from our pre-embedded DB test days.